audio_to_text_summarization_using_openai/main.bal

```diff
@@ -55,9 +55,13 @@ public function main(string podcastURL) returns error? {
     // Summarizes the text using OpenAI text completion API
     text:Client openAIText = check new ({auth: {token: openAIToken}});
     text:CreateCompletionResponse completionRes = check openAIText->/completions.post(textCompletionReq);
-    string summerizedText = check completionRes.choices[0].text.ensureType();
-    io:println("Summarized text: ", summerizedText);
 
+    string? summerizedText = completionRes.choices[0].text;    
+    if summerizedText !is string {
```

Comment:
check whether this is a good practice, why not check directly whether `summerizedText is null`

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/59/commits/a3bb18b9f59a83acd7f228caba3b1a4f59c841d4.